src/actors/realtime_update_actors.rs

```diff
@@ -124,11 +157,13 @@ impl RealTimeReloader {
 
     fn apply_rt(&self, dataset: Arc<Dataset>) -> Result<(), Error> {
         //TODO: make this async
-        let gtfs_rt = fetch_gtfs_rt(&self.gtfs_rt_url)?;
-
-        // we compute the new timetable
-        let rt_dataset = self.make_rt_dataset(dataset, gtfs_rt)?;
+        let gtfs_rts = self
+            .gtfs_rt_urls
+            .iter()
+            .filter_map(|url| fetch_gtfs_rt(&url)) // All failing will be logged in fetch_gtfs
```

Comment:
hum a bit minor too, but I think I prefer if `fetch_gtfs_rt` still return an error and we do

`fetch_gtfs_rt(&url).map_err(|e| log...).ok()`